controller/iteration_test.go

```diff
@@ -193,8 +193,8 @@ func (rest *TestIterationREST) TestSuccessUpdateIterationWithWICounts() {
 				workitem.SystemState:     workitem.SystemStateNew,
 				workitem.SystemIteration: itr.ID.String(),
 			}, uuid.NewV4())
-		require.NotNil(t, wi)
 		require.Nil(t, err)
+		require.NotNil(t, wi)
```

Comment:
@baijum no, but I is not specific to this PR but the test was simply wrong. One has to first check for errors before checking for an arbitrary object not being `nil`.